assets/js/components/surveys/CurrentSurvey.js

```diff
@@ -267,6 +269,8 @@ export default function CurrentSurvey() {
 					choices={ currentQuestion.question.answer_choice }
 					dismissSurvey={ dismissSurvey }
 					question={ currentQuestion.question_text }
+					min_choices={ currentQuestion.min_choices }
+					max_choices={ currentQuestion.max_choices }
```

Comment:
As commented on other PR, this part was not mentioned in AC or IB https://github.com/google/site-kit-wp/pull/3771/files#r684347927

I would just have a switch statement that chooses which component to render rather than have this `SurveyQuestionComponent`